package.json

```
@@ -5,16 +5,17 @@
   "version": "5.0.0",
   "homepage": "https://github.com/SU-SWS/decanter",
   "license": "MIT",
-  "srcDir": "./core/src/",
-  "distDir": "./core/dist/",
-  "kssSrcDir": "./kss/builder/decanter/",
-  "kssDistDir": "./styleguide/",
+  "srcDir": "core/src/",
+  "distDir": "core/dist/",
+  "kssSrcDir": "kss/builder/decanter/",
+  "kssDistDir": "styleguide/",
   "scripts": {
     "build": "webpack --mode=development",
     "dist": "webpack --mode=production",
     "styleguide": "kss --config=kss/kss-config.json",
     "watch": "webpack --mode=development --watch",
-    "dev": "npm run build && npm run styleguide && npm run watch"
+    "start": "webpack-dev-server --open",
```

```suggestion
    "start": "webpack-dev-server --mode=development --open",
```
Adding the mode will suppress the warning about mode not being set.